styles.css

```diff
@@ -29,7 +33,9 @@
 
 .bootstrap-name {
     color: #9B51E0;
+    font-weight:700;
 }
+
```

Comment:
Humm, acho que podemos dividir melhor este .css. Olha só:
- Tenha uma div para cada grande bloco. Faça ela ocupar toda largura da página  e com um tamanho vertical máximo.
- Para a maioria dos casos, você pode apenas centralizar os elementros na div, já vai dar o efeito que queremos ;D
- Dentro de cada DIV haverá um conteúdo específico, se for possível generalizar, declare tamanho e tipo de fonte, etc. etc. etc.

- Para variações, crie tags específicas^^
Por fim, a div controla a posição dos seus filhos, e os filhos devem apenas ser estilizados cada um ao seu modo